edgexfoundry/app-functions-sdk-go
# fix: Use correct metric instance when reporting PipelineMessagesProcessed

fixes #1196

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-functions-sdk-go/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

      <link to docs PR>

## Testing Instructions
Run non-secure EdgeX stack with Device Virtual
Build and run app template locally with only PipelineMessagesProcessed enabled.
Run ASC with metrics-influxdb profile locally with DEBUG logging.
```
./app-service-configurable -p metrics-influxdb | grep "Transformed Metric"
```
Verify the PipelineMessagesProcessed is a simple counter metric rather than a timer metric
```
'PipelineMessagesProcessed,service=new-app-service,pipeline=default-pipeline counter-count=152 1666637357546351500\n'
```
## New Dependency Instructions (If applicable)
